Auth.PollingPublisher/PollingPublisherService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Auth.Infrastructure;
+using Core;
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+
+namespace Auth.PollingPublisher;
+
+public class PollingPublisherService : IHostedService, IDisposable
```

Comment:
You can also inherit `BackgroundService`

Reply:
I would also consider using Hangfire, popular lib within .NET community

Reply:
Good idea, for a real implementation scheduled jobs with hangfire would fit great here